docs/source/concepts/application_usage.rst

```diff
@@ -0,0 +1,58 @@
+Application utilities
+=====================
+
+``HookMan`` offers a few utilities to help with the task to manage and handle plugins used by the application.
+
+
+Install Plugin
+---------------
+
+The method ``Install Plugin`` requires two arguments:
```

Comment:
method `Install Plugin`? I think this description and the rest should be part of the docstring of the related methods, not a separate page (they will get out of sync quickly).

Reply:
This comment applies for this entire page, I think